pkg/transforms/batch.go

```diff
@@ -145,9 +145,9 @@ func (batch *BatchConfig) Batch(edgexcontext *appcontext.Context, params ...inte
 			for {
 				select {
 				case <-batch.done:
-					edgexcontext.LoggingClient.Debug("Batch count has been reached")
+					ctx.LoggingClient().Debug("Batch count has been reached")
 				case <-time.After(batch.parsedDuration):
-					edgexcontext.LoggingClient.Debug("Timer has elapsed")
+					ctx.LoggingClient().Debug("Timer has elapsed")
 				}
 				break
```

Comment:
i don't understand this `break` here.  if we are just wait for select once why we are using for-and then break??

Reply:
Good catch, I will remove the `for` and verify unit tests still pass.

Reply:
Nope, can't do that. The Attribution.txt includes all dependencies in the dependency tree including indirect deps.